server/lib/pbench/indexer.py

```diff
@@ -2507,7 +2511,10 @@ def __init__(self, ptb, iteration, sample, host, tool):
             iterseqno = -1
         itername = iteration
 
-        self.iteration_metadata = _dict_const(name=itername, number=iterseqno,)
+        self.iteration_metadata = _dict_const(
+            name=itername,
+            number=iterseqno,
+        )
```

Comment:
Consider removing the trailing comma and see if Black leaves it all on one line:
`
        self.iteration_metadata = _dict_const(name=itername, number=iterseqno)
`